components/request_otr/browser/request_otr_page.h

```diff
@@ -0,0 +1,62 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_REQUEST_OTR_BROWSER_REQUEST_OTR_PAGE_H_
+#define BRAVE_COMPONENTS_REQUEST_OTR_BROWSER_REQUEST_OTR_PAGE_H_
+
+#include <memory>
+#include <string>
+
+#include "base/values.h"
+#include "components/security_interstitials/content/security_interstitial_page.h"
+#include "url/gurl.h"
+
+namespace content {
+class WebContents;
+}  // namespace content
+
+namespace security_interstitials {
+class SecurityInterstitialControllerClient;
+}  // namespace security_interstitials
+
+namespace request_otr {
+
+// RequestOTRBlockingPage is the interstitial page which will be shown when the
+// browser blocks a top-level, first-party request. A proceed button is
+// provided in the page, along with a checkbox to bypass this interstitial in
+// the future.
+class RequestOTRBlockingPage
```

Comment:
please update filename to match class name

Reply:
OK